setup.cfg

```diff
@@ -43,10 +43,13 @@ install_requires =
 [options.extras_require]
 testing =
     pytest
+    pytest-qt
+    qtbot
```

Comment:
```suggestion
```

afaik qtbot is a fixture provided by pytest-qt, qtbot seems to be a strange stub package with no stars/forks and at 0.0.1
https://pypi.org/project/qtbot/